lib/board.rb

```diff
@@ -0,0 +1,50 @@
+class Board
+  attr_accessor :cells
+
+  @cells = []
+  def initialize
+    #@cells = Array.new(9," ")
+    reset!
+  end
+
+  def reset!
+    @cells = Array.new(9," ")
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts '-----------'
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts '-----------'
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(position)
+    @cells[position.to_i - 1]
+   
+  end
+
+  def full?
+    !(@cells.any? {|cell| cell == " "}) 
```

Comment:
Use the most expressive and efficient iterator for a problem. `any?` is a pretty good choice, but I think we can do better. `Board#full?` is asking if all elements are not empty strings. You can use logic that is symmetrical to those semantics with the [`#all?`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-all-3F) iterator.

`@cells.all? {|cell| cell != " "}`

Sometimes, having a method that returns `true` for a certain situation is more readable than using `!` with a method that returns `false`.